src/test/e2e-emulated/contracts/intrinsics.tact

```diff
@@ -6,6 +6,10 @@ contract IntrinsicsTester {
     d: Cell = cell("te6cckEBAQEADgAAGEhlbGxvIHdvcmxkIXgtxbw=");
     e: Int = pow(2, 9);
     f: Int = sha256("hello world");
+    g: Slice = slice("te6cckEBAQEADgAAGEhlbGxvIHdvcmxkIXgtxbw=");
+    h: Slice = rawSlice("hello world");
+    i: Int = ascii("hello world");
+    j: Int = crc32("hello world");
```

Comment:
Let's reuse the test cases from the TON docs (https://docs.ton.org/develop/func/literals_identifiers#string-literals). For instance:

> `"transfer(slice, int)"c` becomes 2235694568 int const